setup.py

```diff
@@ -34,7 +34,7 @@ def main():
         test_requirements.append('unittest2')
     setup(
         name='genty',
-        version='1.2.2',
+        version='1.2.3',
```

Comment:
The fix for #43 could, in theory, cause someone's tests to break (if for some reason their test runner breaks on the particular character you chose to use). Should we bump the minor version (so go to '1.3.0')?